tests/autoyast/installation.pm

```diff
@@ -16,6 +16,7 @@
 use strict;
 use base 'basetest';
 use testapi;
+use utils qw(clear_console);
 
```

Comment:
I can't find any reference to python on https://en.wikipedia.org/wiki/Domain-specific_language - but 4 for perl.